besu/src/test/java/org/hyperledger/besu/RunnerTest.java

```diff
@@ -393,6 +345,24 @@ private void syncFromGenesis(final SyncMode mode, final GenesisConfigFile genesi
     }
   }
 
+  private int getNumber(final Response response) throws IOException {
+    final int currentBlock =
+        UInt256.fromHexString(new JsonObject(response.body().string()).getString("result"))
+            .intValue();
+    return currentBlock;
+  }
+
+  @NotNull
```

Comment:
what is @NotNull accomplishing in this case?  it is a jetbrains annotation, and won't be checked by findbugs or CI AFAIK.  Also this is on a test method that returns the result of a builder.  

I ask everytime I see the jetbrains import.

Reply:
👍 good point. removed